engine/src/main/java/org/terasology/rendering/dag/nodes/ShadowMapNode.java

```diff
@@ -46,7 +50,7 @@
  * - https://docs.google.com/drawings/d/13I0GM9jDFlZv1vNrUPlQuBbaF86RPRNpVfn5q8Wj2lc/edit?usp=sharing
  */
 public class ShadowMapNode extends AbstractNode {
-    private static final String SCENE_SHADOW_MAP = "sceneShadowMap";
+    public static final ResourceUrn SHADOW_MAP_URN = new ResourceUrn("engine:sceneShadowMap");
```

Comment:
_mumbles about how we will eventually allow for multiple shadow maps_ - but nothing to worry about now! :smile: